target/linux/ath79/dts/qca9531_comfast_cf-wr752ac-v1.dts

```diff
@@ -0,0 +1,144 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "qca953x.dtsi"
+
+/ {
+	compatible = "comfast,cf-wr752ac-v1", "qca,qca9531";
+	model = "COMFAST CF-WR752AC v1";
+
+	aliases {
+		serial0 = &uart;
+		led-boot = &led_lan;
+		led-failsafe = &led_lan;
+		led-upgrade = &led_lan;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		pinctrl-names = "default";
+		pinctrl-0 = <&jtag_disable_pins &led_wlan2g_pin &led_wlan5g_pin>;
+
+		led_lan: lan {
+			label = "cf-wr752ac-v1:blue:lan";
+			gpios = <&gpio 12 GPIO_ACTIVE_LOW>;
+			linux,default-trigger = "netdev";
+		};
+
+		wlan2g {
+			label = "cf-wr752ac-v1:green:wlan2g";
+			gpios = <&gpio 15 GPIO_ACTIVE_LOW>;
+			linux,default-trigger = "phy1tpt";
+		};
+
+		wlan5g {
+			label = "cf-wr752ac-v1:red:wlan5g";
+			gpios = <&gpio 11 GPIO_ACTIVE_LOW>;
+			linux,default-trigger = "phy0tpt";
+		};
+	};
+
+	keys {
+		compatible = "gpio-keys";
+
+		button {
+			label = "reset";
+			linux,code = <KEY_RESTART>;
+			gpios = <&gpio 17 GPIO_ACTIVE_LOW>;
+			debounce-interval = <60>;
+		};
+	};
+
+	watchdog {
+		compatible = "linux,wdt-gpio";
+
+		gpios = <&gpio 13 GPIO_ACTIVE_LOW>;
+		hw_algo = "toggle";
+		hw_margin_ms = <1200>;
+		always-running;
+	};
+};
+
+&pinmux {
+	led_wlan2g_pin: pinmux_lan {
+		pinctrl-single,bits = <0x0c 0x0 0xff000000>;
+	};
+
+	led_wlan5g_pin: pinmux_wlan2g {
+		pinctrl-single,bits = <0x08 0x0 0xff000000>;
+	};
+};
+
+&pcie0 {
+	status = "okay";
+};
+
+&spi {
+	status = "okay";
+	
+	num-cs = <1>;
+
+	flash@0 {
+		compatible = "winbond,w25q128", "jedec,spi-nor";
```

Comment:
Do you need the "winbond,w25q128" compatible?

Reply:
used flash here is w25q128, 
precedent: qca9533_comfast_cf-e110n-v2.dts

Reply:
And that's the only in entire ath79. "jedec,spi-nor" will provide auto-identification, so I will answer my question myself: You don't need it, please remove it and just keep "jedec,spi-nor".

Reply:
OK. Removed.